### ISSUE TITLE
Support for other SQL DBs

### ISSUE THREAD

**Issue by gokceneraslan** [Element-Research/hypero#2]
It would be nice to be able to use other types of SQL databases, maybe using [keplerproject/luasql](https://github.com/keplerproject/luasql). Using `sqlite`, for example, would decrease the configuration overhead for people who do not want to configure/modify/start an SQL server.

**Comment by nicholas-leonard** [comment]
Yeah sqlite would be a nice addition. However, I don't have time to implement it myself right now :)
